open-craft/custom-student-verification
# [NELP-13] Fix RTL direction issue and translations

## Description

This PR fixes content direction when using an RTL language. Also, it fixes the broken translations.

## Supporting information

https://tasks.opencraft.com/browse/NELP-13

**Screenshot**
<img width="1390" alt="Screenshot 2021-04-09 at 8 14 56 AM" src="https://user-images.githubusercontent.com/1010244/114120919-466c6480-990f-11eb-9fc0-018ffc7827b3.png">

## Testing instructions

1. Clone this PR on your local devstack - `src/` folder
2. Install the app on your LMS - `make lms-shell` then `pip install -e /edx/src/custom-student-verification`.
3. Restart LMS - `make lms-restart`
4. Change the language to `ar` from `http://localhost:18000/update_lang/`
5. Visit `http://localhost:18000/verify_student/reverify` to check if the content direction is RTL for the form.

## Deadline

ASAP

## Other information

We don't have `ar` translations for all strings used in this app.

**Reviewer**
